src/app/components/Curation/index.tsx

```diff
@@ -49,7 +49,6 @@ export default ({
   title = '',
   topStoriesTitle = '',
   link = '',
-  headingLevel = 2,
```

Comment:
Let's also update the interface for the Curation component, since we no longer require `headingLevel` it to be passed in as a prop

Reply:
I have removed headingLevel from CurationGridProps

Reply:
I think you need to keep the headingLevel on the CurationGridProps, but remove it from the Curation props.. https://github.com/bbc/simorgh/blob/latest/src/app/models/types/curationData.ts#L65

Reply:
done